research/CongressKeynote/Congress-Keynote.md

```diff
@@ -0,0 +1,466 @@
+
+For convenience, we refer to these spectrograms as long-duration, false-colour spectrograms or LDFC spectrograms.
+
+Different combinations of indices give different views of the soundscape. Note how less acoustic structure is visible in the bottom spectrogram because two of the three indices are correlated. So here is an important point - the three indices used to construct an LDFC spectrogram should be minimally correlated. (For more detail see previous tutorials and publications.)
+
+---
+
+## Slide 11: Biophony revealed in a 24-hour LDFC spectrogram
+{% include figure.html src="Slides/Slide11.png" caption="Slide 11." %}
+
+In this slide, we identify some of the birds that produced the acoustic events visible in the LDFC spectrograms. A cicada chorus is visible at 800Hz, 6pm. Note that much of the structure in these images comes about because a bird (or birds) tends to sit in the same tree for a few minutes and is likely to repeat the same song/call. Even though a bird may only call once or twice in a minute, one or other acoustic index will “detect” the call and calls over consecutive minutes will leave a clearly visible trace in the spectrogram. Note that LDFC spectrograms were never designed to identify bird species. The fact that this is possible has turned out to be remarkably useful. More on this later.
+
+You can see an interactive version of this slide [here](http://www.ecosounds.org/Zoom/1-top). 
+
+---
+
+## Slide 12: LDFC spectrograms of three sites at different latitudes
+{% include figure.html src="Slides/Slide12.png" caption="Slide 12." %}
+
+This slide compares three 24-hour, false-colour spectrograms of three soundscapes from different latitudes. All these recordings were obtained in the first week of July (winter) 2015. The top PNG recording is dominated by insects. It is hard to imagine stuffing more sound into this soundscape. PNG jungles are very loud. The middle recording (from Gympie National Park, about 100 kilometers north of Brisbane) is very quiet at night (because the temperatures are near freezing) and dominated by birds during the day. The bottom recording is from Sturt National Park in the far north-west corner of New South Wales, Australia. The conditions are desert-like. Not only is it very cold at night, but during the day it blows a gale (the yellow-red lines in the middle of the spectrogram). The desert morning chorus is diminutive. *Soundmarks* are very useful (actually essential) in order to orient oneself in a soundscape. They are the equivalent of *landmarks* in a landscape.
```

Comment:
Do you want to identify the top recording as a recording from Papua New Guinea, or leave it ambiguous?
100 kilometers, closer to 150 kilometers